controllers/service/eventhandlers/service_events.go

```diff
@@ -14,6 +14,7 @@ import (
 )
 
 const loadBalancerTypeNLBIP = "nlb-ip"
+const LoadBalancerTypeNLBInstance = "nlb-instance"
```

Comment:
This will not work very well with the in-tree service controller for service type LoadBalancer. The in-tree controller will end up provisioning a CLB for the service.

The path we want to follow is to use the load balancer type 'external' so that the in-tree code ignores the service object, and have an additional annotation to determine the instance mode.

Reply:
Thank you for the feedback. We are on 1.18 so we are using type `NodePort`. To clarify the path forward, you are suggestion the `service.beta.kubernetes.io/aws-load-balancer-type` annotation be set to `external` and another annotation to define instance mode, either `nlb-ip` or `nlb-instance`?

Reply:
We are also working on the instance mode support and targeting it post reInvent 2020.